test-install.js

```diff
@@ -85,3 +90,14 @@ function mockPlatform(newPlatformValue) {
         value: newPlatformValue
     });
 }
+/**
+ * Helper for Chromium installation tests.
+ * @param t Test engine API object.
+ */
+async function installChromeAndVerify(t) {
```

Comment:
Hey Dmytro, I think we've come full circle - with the new tests I wrote I don't think we need the "install from mirror" test any more - the existing install test verifies that it can download and install the chromium binaries. 

The new tests verify that the URL can be changed to a mirror, that's really what we care about.

The "install from mirror" test is superfluous now, all it is doing is testing the actual mirror itself, not "node-chromium".

Reply:
Agree!